_integration-schemas/trello/boards.md

```diff
@@ -0,0 +1,264 @@
+---
+tap: "trello"
+version: "1"
+key: ""
+
+name: "boards"
+doc-link: "https://developer.atlassian.com/cloud/trello/rest/#api-boards-id-get"
+singer-schema: "https://github.com/singer-io/tap-trello/blob/master/tap_trello/schemas/boards.json"
+description: |
+  The {{ table.name }} table contains information about your boards idividual boards in {{ integration.display_name }}.
+
+replication-method: "Key-based Incremental"
+
+api-method:
+    name: "Get a Board"
+    doc-link: "https://developer.atlassian.com/cloud/trello/rest/#api-boards-id-get"
```

Comment:
This actually isn't the endpoint we're using to get this data. We're using what's in `endpoint`, here:

```python
class Boards(Unsortable, Stream):
    stream_id = "boards"
    stream_name = "boards"
    endpoint = "/members/{}/boards"
    key_properties = ["id"]
    replication_method = "FULL_TABLE"
```

https://github.com/singer-io/tap-trello/blob/master/tap_trello/streams.py#L326

Which matches up with [this endpoint in the Trello docs](https://developer.atlassian.com/cloud/trello/rest/#api-members-id-boards-get).

**This is very different than just getting a single board.** What does the structure of this endpoint (`/members/{id}/boards`) tell you? How will it affect the rest of the data that a user can get from the integration? What should the user setting up the integration do prior to creating the integration in Stitch?

Let's chat about this if you're not sure, but it has implications on the setup prerequisites and the data a single user can get for not only this table, but every table in the integration.